### PR TITLE
First draft for CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Vogel612**

### CODE REVIEW

**Comment by Vogel612** [comment]
See #3239

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- Show and deserve respect
+- Be nice
+- We don't bite :)
+- Assume people's best intentions unless proven otherwise
+
+## How can I contribute?
+
+### Bug Reports
+
+The development team is always more or less happy to hear about bugs. Reported bugs are an opportunity to make Rubberduck better.
```
Comment (retailcoder) [inline CONTRIBUTING.md]: More or less happy? Nah, *very happy*!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+In that room the core team talks about the duck and whatever else comes up.
+N.B.: The rules of the Stack Exchange Network apply to everything you say in that room. But basically those are the same rules as those in the [CoC](#Code_of_Conduct).
+
+### Translations
+
+Rubberduck is Localized in multiple languages.
+All these translations have been provided by volunteers.
+We welcome both new translations as well as improvements to current translations very much.
+
```
Comment (retailcoder) [inline CONTRIBUTING.md]: Link to your awesome translation tool?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+Then there's \[difficulty-03-duck\] which requires a good handle on C# and at least an idea of how the components of Rubberduck play toghether.
+And finally there's \[difficulty-04-quackhead\] for when you really want to bang your head against the wall.
+They require both good knowledge of C# and a deep understanding of how Rubberduck works.
+
+In addition to this, a few people have put together some helpful resources about how Rubberduck works internally and help contributors tell left from right in the ~250k Lines of Code.
+
+You can find their work in the [github wiki](https://github.com/rubberduck-vba/Rubberduck/wiki). If something in there doesn't quite seem to match, it's probably because nobody got around to updating it yet. You're very welcome to **improve** the wiki by adding and correcting information. (see right above)
+
+In case this doesn't quite help you or the information you need hasn't been added to the wiki, you can **always** ask questions.
+Please do so in the ["War room"](https://chat.stackexchange.com/rooms/14929).
```
Comment (retailcoder) [inline CONTRIBUTING.md]: Questions could also be asked in a [support] issue, or on Stack Overflow, appropriately tagged with [rubberduck] if the question is specific to RD and best answered by people that know its internal API.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+Note that you will need an account on any of the Stack Exchange sites with at least 20 reputation points to ask questions there.
```
Comment (retailcoder) [inline CONTRIBUTING.md]: One way to quickly earn the reputation is to submit your PR (e.g. a new inspection) on Code Review Stack Exchange, tagged with [rubberduck].

File: CONTRIBUTING.md (hunk 3 above)
Comment (MDoerner) [inline CONTRIBUTING.md]: I think you actually cannot put the [support] label on an issue unless you are in the dev group.

**Comment by Hosch250** [comment]
LGTM.

File: CONTRIBUTING.md (hunk 3 above)
Comment (retailcoder) [reply CONTRIBUTING.md]: Aye, but we'll tag support questions as such =)

**Pull request merged**